swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/exclusions/ExclusionPolicy.kt

```diff
@@ -36,8 +36,8 @@ interface ExclusionPolicy {
 
   fun getType(): ExclusionType
 
-  fun notWhitelistedMessage(value: String?, patterns: Set<Any> = emptySet()): String {
-    return "$value not whitelisted. Matched: ${patterns.joinToString(",")}"
+  fun notAllowlistedMessage(value: String?, patterns: Set<Any> = emptySet()): String {
+    return "$value not allowed. Matched: ${patterns.joinToString(",")}"
```

Comment:
`not in allowlist`? `not allowed` feels like it could also be a validation type error

Reply:
Good call.